packages/indexer-common/src/indexing-status.ts

```diff
@@ -16,6 +16,14 @@ export interface SubgraphDeploymentAssignment {
   node: string
 }
 
+export interface SubgraphFeatures {
+  // `null` is only expected when there are errors in the Subgraph Manifest.
+  network: string | null
+  // manifest validation errors
+  errors: string[]
+  features: 'nonFatalErrors' | 'grafting' | 'fullTextSearch' | 'ipfsOnEthereumContracts'
```

Comment:
It doesn't look like the `errors` and `features` properties are ever used?

Reply:
Positive.
We don't use those fields in the rest of the code.
Removed them in e1b4e9a1.